src/plots/gl2d/scene2d.js

```diff
@@ -487,7 +487,11 @@ proto.draw = function() {
                 this.lastPickResult.dataCoord[0] !== nextSelection.dataCoord[0] ||
                 this.lastPickResult.dataCoord[1] !== nextSelection.dataCoord[1])
             ) {
-                var selection = this.lastPickResult = nextSelection;
+                var selection = nextSelection;
+                this.lastPickResult = {
+                    trace: nextSelection.trace, // could we just retain/compare the trace uid?
```

Comment:
@etpinard Yes it works. Made the change, checked tooltip and CPU behavior (all OK),  pushed.